macros/utils/cross_db_utils/timediff.sql

```diff
@@ -3,19 +3,12 @@
     {{ return(adapter.dispatch('timediff', 'elementary')(timepart, first_timestamp, second_timestamp)) }}
 {%- endmacro %}
 
-{# Snowflake #}
+{# For Snowflake, Databricks, Redshift, Postgres & Spark #}
+{# the dbt adapter implementation supports both timestamp and dates #}
 {% macro default__timediff(timepart, first_timestamp, second_timestamp) %}
-    datediff({{ timepart }}, {{ first_timestamp }}, {{ second_timestamp }})
+    {{ elementary.datediff(first_timestamp, second_timestamp, timepart)}}
```

Comment:
Is it working for postgres?
It has different implementation from the rest (not sure if dbt.datediff) handles that for us or not so I am asking.

Reply:
Yes, checked and it does.